src/main/cpp/controllers/turret/ShooterSaveState.h

```diff
@@ -49,7 +48,7 @@ class ShooterSaveState : public IState
     double degreesToTurn;
 
     private:
-
+    ControlData*   m_controlData;
     IMechanism*    m_turret;
     ControlData*   m_control;
     double         m_target;

```

Comment:
this isn't getting set in the cpp file, so your comparison in AtTarget is going to be random